content-src/components/PocketStories/PocketStories.js

```diff
@@ -59,7 +59,7 @@ const PocketStories = React.createClass({
 
     return (<section className="pocket-stories spotlight">
       <h3 className="section-title">
-        <FormattedMessage id="header_stories" />
+        Trending Stories By
```

Comment:
From #2463, it sounds like this will potentially be shown to any user including non-en-US, so we would want this localized.